dtfs-central-api/src/external-api/send-email.ts

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable @typescript-eslint/no-unsafe-return */
```

Comment:
We shouldn't be using eslint disables

Reply:
this is a problem across the service and cannot be fixed now

Reply:
Ah ok, what is the problem?

Reply:
removed return and lint disable